### PR TITLE
Add ListWorkflowJobsAttempt method to ActionsService

### LINKED ISSUE #3059: Add support for  Actions list jobs for a workflow run attempt
I am reaching out to inquire about support for the API. It appears that the API for `list jobs for a workflow run attempt` has not been implemented yet. I was wondering if it would be possible to consider adding this feature. docs: https://docs.github.com/en/rest/actions/workflow-jobs?apiVersion=2022-11-28#list-jobs-for-a-workflow-run-attempt

Thank you very much for your attention and assistance. I look forward to any updates you might have on this matter.

### PR DESCRIPTION
**Pull request by fchimpan**
Fixes: #3059.

Add support ListWorkflowJobsAttempt API. 
https://docs.github.com/rest/actions/workflow-jobs#list-jobs-for-a-workflow-run-attempt

### CODE REVIEW

File: github/actions_workflow_jobs_test.go
Comment (gmlewis) [inline github/actions_workflow_jobs_test.go:130]:
```suggestion

	const methodName = "ListWorkflowJobsAttempt"
	testBadOptions(t, methodName, func() (err error) {
		_, _, err = client.Actions.ListWorkflowJobsAttempt(ctx, "\n", "\n", -1, -1, opts)
		return err
	})
	testNewRequestAndDoFailure(t, methodName, client, func() (*Response, error) {
		got, resp, err := client.Actions.ListWorkflowJobsAttempt(ctx, "o", "r", 29679449, 1, opts)
		if got != nil {
			t.Errorf("testNewRequestAndDoFailure %v = %#v, want nil", methodName, got)
		}
		return resp, err
	})
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @fchimpan !
This looks like a great start apart from the code coverage.

I've added a suggestion that should hopefully clear up the coverage tests.

**Review by gmlewis — APPROVED** [review]
Thank you, @fchimpan !
LGTM

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Comment by fchimpan** [comment]
@gmlewis 

Thanks for your advice. I was already working on improving the test coverage before I noticed your comments, so there may have been some overlap. I would appreciate it if you could review the changes. I look forward to your feedback.

Best regards,

**Comment by fchimpan** [comment]
@gmlewis 
Thank you for the comment. I understand and am happy to help. 
I have an interest in this repository, so I will definitely cooperate.

Since I don't have the authority to approve pull requests, should I just leave comments on the PR if I have any suggestions?

**Comment by gmlewis** [comment]
Thank you, @fchimpan !
Yes, feel free to leave comments, and when you are satisfied with the changes, you can say "LGTM" (Looks Good To Me) and I will take that as approval.

**Comment by fchimpan** [comment]
Thank you @gmlewis 
Understood, I will check it out.

By the way, sometimes I come across APIs that are not yet implemented. In such cases, is it alright if I propose a PR like I did this time?

Thank you in advance.

**Comment by gmlewis** [comment]
> By the way, sometimes I come across APIs that are not yet implemented. In such cases, is it alright if I propose a PR like I did this time?

Absolutely!  We welcome all contributions.
The GitHub v3 API is amazingly huge and constantly updating, so we definitely welcome the help.
Thank you!

**Comment by gmlewis** [comment]
Thank you, @valbeat !
Merging.

**Pull request merged**
